tests/jeos/prepare_firstboot.pm

```diff
@@ -47,6 +47,8 @@ sub run {
     if ($is_generalhw_via_ssh) {
         # Do not set eth0 down as we are connected through ssh!
         assert_script_run("sed -i 's/ip link set down \"\$d\" #/if [ \"eth0\" != \"\$d\" ]; then ip link set down \"\$d\"; fi; #/' /usr/lib/jeos-firstboot");
```

Comment:
Why not just delete any occurence of `ip link set down`?

Reply:
It may be useful for other interfaces, no?

Reply:
It only works on ethernet interfaces, so except for `eth0` there isn't anything (unless you have some USB ethernet controller)

Reply:
My goal is to be less invasive as possible. And you may have multiple Ethernet devices, or even `can` devices. You probably want them down after `jeos-firstboot` played with it.

Reply:
Let's try to just ignore the `ip link set down` command.